tests/ha/single_pod_restart/test_data_server_pod_restart_api.py

```diff
@@ -310,3 +323,134 @@ def test_chunk_upload_during_data_server_pod_restart(self):
                                       f"Actual checksum: {dnld_chksm}")
         LOGGER.info("Step 8: Successfully downloaded object and verified checksum")
         LOGGER.info("COMPLETED: Verify chunk upload during 1 data pod and 1 server pod restart")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-45533")
+    def test_obj_overwrite_during_data_server_pod_restart(self):
+        """
+        Verify object overwrite during 1 data pod and 1 server pod restart
+        """
+        LOGGER.info("STARTED: Verify object overwrite during 1 data pod and 1 server pod restart")
+        file_size = HA_CFG["5gb_mpu_data"]["file_size"]
+        output = Queue()
+        event = threading.Event()
+        LOGGER.info("Step 1: Create bucket %s and perform upload of object size %s MB and "
+                    "Overwrite the object", self.bucket_name, file_size)
+        s3_data = {self.bucket_name: [self.object_name, file_size]}
+        resp = self.ha_api.object_overwrite_dnld(self.s3_test_obj, s3_data, iteration=1,
+                                                 random_size=False)
+        assert_utils.assert_true(resp[0], "Failure observed in overwrite method.")
+        upld_chcksm = list(resp[1].values())[0][0]
+        for checksum in resp[1].values():
+            assert_utils.assert_equal(checksum[0], checksum[1],
+                                      f"Checksum doesn't match, Expected: {checksum[0]} "
+                                      f"Received: {checksum[1]}")
+        LOGGER.info("Step 1: Create bucket %s and perform upload of object size %s MB and "
```

Comment:
we can login instead as: %s bucket created and upload and overwrite of object %s is done successfully. 
This same logger got me confused